tests/manual-test-cases/Group5-Functional-Tests/5-3-Enhanced-Linked-Mode.robot

```diff
@@ -132,6 +112,6 @@ Test
     Set Environment Variable  TEST_RESOURCE  cls
     Set Environment Variable  TEST_TIMEOUT  30m
     
-    Install VIC Appliance To Test Server  certs=${true}  vol=default
+    Install VIC Appliance To Test Server
```

Comment:
This defaults to cert=true, and saw some issue with VIC install on nimbus when certs were enabled. Doing a quick run to make sure its ok with the latest. LGTM otherwise.